WcaOnRails/app/views/regional_organizations/_regional_organization_form.html.erb

```diff
@@ -33,17 +33,21 @@
     <%= link_to(image_tag(@regional_organization.extra_file.preview(resize: "200x200")), rails_blob_path(@regional_organization.extra_file, disposition: "attachment"))  %>
   <% end %>
 
-  <div class="form-group">
-    <div class="col-sm-offset-2 col-sm-9">
-      <div class="row">
-        <%= f.input :start_date, as: :date_picker, wrapper: :ranged_datetime %>
-        <%= f.input :end_date, as: :date_picker, wrapper: :ranged_datetime %>
+  <% if @current_user.can_manage_regional_organizations? %>
```

Comment:
Correct me if I'm wrong, but it looks like this is the only restriction on `start_date` and `end_date`, so a direct request with these fields would set them regardless user permissions. The solution would be modifying `regional_organization_params` to conditionally include `start_date` and `end_date` for board members.

Reply:
Thanks for the catch, I'm updating this too!